lib/iam_ssoe_oauth/session_manager.rb

```diff
@@ -43,6 +36,8 @@ def destroy_user_session
       end
     end
 
+    private
```

Comment:
Will making these private have any effect on IAM SSOe session metrics in config/initializers/statsd.rb? It might not, I just saw that it is referenced there and want to make sure.

Reply:
That's a good question. Looking at the statsd gem source those config calls use meta programming to add in the statsd calls, which aren't blocked by being private. There's (passing 😄) [specs](https://github.com/department-of-veterans-affairs/vets-api/blob/mobile-api-logout-fix/modules/mobile/spec/controllers/application_controller_spec.rb#L108) that check's if the statsd calls are triggered.